envs/conda_build_config.yaml

```diff
@@ -1,5 +1,6 @@
 abseil_cpp:
-  - 20200225
+  - 20200225 # [not s390x]
+  - 20200923 # [s390x]
```

Comment:
Can you change this one to `20200923` across the board (unqualified) as well. This will be needed to build Arrow with pb3.14 on the other platforms as well.

Reply:
ok